worlds/tracker/TrackerClient.py

```diff
@@ -150,6 +150,25 @@ def _cmd_reset_ignored(self):
         self.ctx.ignored_locations.clear()
         updateTracker(self.ctx)
         logger.info("Reset ignored locations.")
+    def _cmd_next_progression(self):
+        """Finds all items that will unlock a check immediately when collected, and how many new checks they will unlock."""
```

Comment:
UT can be incorrect in its assumptions, so you should mention that
```suggestion
        """Finds all items that will unlock a check immediately when collected, and a best guess of how many new checks they will unlock."""
```